src/Umbraco.Core/Models/PropertyTypeCollection.cs

```diff
@@ -30,7 +30,7 @@ public PropertyTypeCollection(bool supportsPublishing, IEnumerable<IPropertyType
     // This baseclass calling is needed, else compiler will complain about nullability
 
     /// <inheritdoc />
-    public bool IsReadOnly => ((ICollection<IPropertyType>)this).IsReadOnly;
+    public bool IsReadOnly => false;
```

Comment:
It's on purpose because this line is a circular reference, so the expected behaviour here is undefined. Besides that, there were no references in the code to this property according to visual studio. It also causes visual studio to crash when observing objects of this type while debugging. False is the default for a boolean, so that seemed the most obvious choice.

I also checked if the base class had an implementation, but it didn't, so there's nothing to base this value on.

Reply:
And this object implements methods for modifying the collection, so it makes sense that it's not read only

Reply:
Ah great then @JasonElkin , be my guest 😉

Reply:
And apologies @D-Inventor that I lost track of your PR 😅